tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -805,6 +805,16 @@ internal static class FeatureFlags
             /// </summary>
             public const string OpenTelemetryEnabled = "DD_TRACE_OTEL_ENABLED";
 
+            /// <summary>
+            /// Enables beta support for exporting OTLP metrics generated by the OpenTelemetry Metrics API.
+            /// </summary>
+            public const string OpenTelemetryMetricsEnabled = "DD_METRICS_OTEL_ENABLED";
+
+            /// <summary>
+            /// List of meters to add to the metrics exporter for the experimental OpenTelemetry Metrics API support.
+            /// </summary>
+            public const string EnabledMeters = "DD_TRACE_ENABLED_METERS";
```

Comment:
Now that we're going with `DD_METRICS_OTEL_ENABLED`, what do you think of renaming this to something like `DD_METRICS_OTEL_METER_NAMES`?